pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensible/data/BrokerLoadData.java

```diff
@@ -0,0 +1,324 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.loadbalance.extensible.data;
+
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import lombok.Data;
+import org.apache.pulsar.broker.ServiceConfiguration;
+import org.apache.pulsar.common.naming.NamespaceName;
+import org.apache.pulsar.common.naming.TopicDomain;
+import org.apache.pulsar.policies.data.loadbalancer.LocalBrokerData;
+import org.apache.pulsar.policies.data.loadbalancer.NamespaceBundleStats;
+import org.apache.pulsar.policies.data.loadbalancer.ResourceUsage;
+import org.apache.pulsar.policies.data.loadbalancer.SystemResourceUsage;
+
+/**
+ * Contains all the data that is maintained locally on each broker.
+ *
+ * Migrate from {@link org.apache.pulsar.policies.data.loadbalancer.LocalBrokerData}.
+ * And removed the lookup data, see {@link BrokerLookupData}
+ */
+@Data
+public class BrokerLoadData {
+
+    public static final String TOPIC =
+            TopicDomain.non_persistent
+                    + "://"
+                    + NamespaceName.SYSTEM_NAMESPACE
+                    + "/broker-load-data";
+    private static final double gigaBitToByte = 128 * 1024 * 1024.0;
+
+    // Most recently available system resource usage.
+    private ResourceUsage cpu;
+    private ResourceUsage memory;
+    private ResourceUsage directMemory;
+
+    private ResourceUsage bandwidthIn;
+    private ResourceUsage bandwidthOut;
+
+    // Message data from the most recent namespace bundle stats.
+    private double msgThroughputIn;
+    private ResourceUsage msgThroughputInUsage;
+    private double msgThroughputOut;
+    private ResourceUsage msgThroughputOutUsage;
+    private double msgRateIn;
+    private double msgRateOut;
+
+    // Timestamp of last update.
+    private long lastUpdate;
+
+    // The stats given in the most recent invocation of update.
+    private Map<String, NamespaceBundleStats> lastStats;
```

Comment:
Please note that when we serialize this object to JSON(when publishing this object to the `BrokerLoadDataStore`). 

We need to ignore the majority of these parameters (e.g. bundles and lastStats) to minimize the payload size.

Reply:
Updated.